### DISCUSSION TITLE
It's possible to return to shell after launch?

### DISCUSSION THREAD

**Discussion by marcioflaviof** [kovidgoyal/kitty#5856] [Q&A] [answered]
Basically I trying to launch my fish shell with nvim for example, `fish -c 'nvim'` and after I quit nvim return to fish, it's possible?

**Comment by kovidgoyal** [comment, marked as answer]
On Thu, Jan 05, 2023 at 03:34:38AM -0800, Marcio Flavio wrote:
> Basically I trying to launch my fish shell with nvim for example, `fish -c 'nvim'` and after I quit nvim return to fish, it's possible?

fish -c 'nvim; fish'
